src/v2/cookbook/adding-instance-properties.md

```diff
@@ -106,34 +107,40 @@ In case you're not aware, methods added to a prototype in JavaScript gain the co
 
 Let's take advantage of this in a `$reverseText` method:
 
-``` js
-Vue.prototype.$reverseText = function (propertyName) {
-  this[propertyName] = this[propertyName].split('').reverse().join('')
+```js
+Vue.prototype.$reverseText = function(propertyName) {
+  this[propertyName] = this[propertyName]
+    .split('')
+    .reverse()
+    .join('')
 }
 
 new Vue({
   data: {
     message: 'Hello'
   },
-  created: function () {
-    console.log(this.message)    // => "Hello"
+  created: function() {
+    console.log(this.message) // => "Hello"
     this.$reverseText('message')
-    console.log(this.message)    // => "olleH"
+    console.log(this.message) // => "olleH"
```

Comment:
It looks like these will were probably automatically formatted by Prettier. We've previously followed the StandardJS conventions in the Vue docs, but I've actually been thinking about moving towards Standard + Prettier overall, with linting built into the project so this is probably OK.